TicketManagmentSystem.Api/TicketManagmentSystem.Api/Controllers/EventController.cs

```diff
@@ -1,36 +1,78 @@
-﻿using Microsoft.AspNetCore.Http;
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.IdentityModel.Tokens;
+using System.ComponentModel;
+using TicketManagmentSystem.Api.Models;
 using TicketManagmentSystem.Api.Models.Dto;
+using TicketManagmentSystem.Api.Repositories;
 
 namespace TicketManagmentSystem.Api.Controllers
 {
     [Route("api/[controller]/[action]")]
     [ApiController]
     public class EventController : ControllerBase
     {
+
+        private readonly IEventRepository eventRepository;
+        private readonly IMapper mapper;
+
+        public EventController(IEventRepository eventRepository, IMapper mapper)
+        { 
+            this.eventRepository = eventRepository;
+            this.mapper = mapper;
+        }
+
+
         [HttpGet]
         public ActionResult<List<EventDto>> GetAll()
         {
-            var events = new List<EventDto>();
+            var events = eventRepository.GetAll();
```

Comment:
It is not recommended to call repository methods directly from controller. Instead, create a new project TicketManagementSystem.BusinessLogic, that contains a class that references the repository. In the controller, call the new class. For more details, see 3 layers architecture: https://medium.com/@deanrubin/the-three-layered-architecture-fe30cb0e4a6